services/notification-service/src/main/java/com/sourcefuse/jarc/services/notificationservice/NotificationServiceApplication.java

```diff
@@ -0,0 +1,38 @@
+package com.sourcefuse.jarc.services.notificationservice;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configurers.CsrfConfigurer;
+import org.springframework.security.web.SecurityFilterChain;
+
+@SpringBootApplication
+@ComponentScan(
+  {
+    "com.sourcefuse.jarc.core",
+    "com.sourcefuse.jarc.services.notificationservice"
```

Comment:
please add authlib package to use auth middlewares

Reply:
will do in separate PR